services/ui-src/src/utils/other/export.tsx

```diff
@@ -136,8 +137,10 @@ export const renderResponseData = (
     : fieldResponseData;
   const missingEntryVerbiage = notApplicable
     ? verbiage.missingEntry.notApplicable
-    : verbiage.missingEntry.noResponse;
-  const missingEntryStyle = notApplicable ? sx.notApplicable : sx.noResponse;
+    : `${verbiage.missingEntry.noResponse}, optional`;
```

Comment:
I think this might be interfering with other fields that are empty. Compare the PDF view of the "Information for Primary Contact" pages

this branch:
![Screenshot 2024-05-03 at 10 15 51 AM](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/57802560/47023383-1ffa-4c8c-bfa2-7eb63ed8a737)

main:
![Screenshot 2024-05-03 at 10 16 26 AM](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/57802560/420d6309-e01a-4ee3-95c9-7a5e5471fdaa)